bin/md-to-html5

```diff
@@ -1,15 +1,180 @@
 #! /bin/sh
 
-template="$0.tmpl.html5"
+HERE=$(cd $(dirname $0); pwd)
+THIS=$(basename $0)
+
+# Our HTML5 template, produced from the default pandoc template with some
+# massaging by 'mk-pandoc-template'.
+template="$HERE/../inc/pandoc-template.html5"
+
+# Use the solarized highlight theme, since that's what we use in the blog
+# See https://pandoc.org/MANUAL.html#syntax-highlighting
+highlightstyle="$HERE/../inc/pandoc-solarized.theme"
+
+######################################################################
+#
+#  Option processing
+#
+
+output=
+index=
+
+shortopts='io:h'
+longopts='index,output:,help'
+usage="\
+Usage 1: $THIS [ options ] [ key=value ... ] < file.md > file
+Usage 2: $THIS [ options ] [ key=value ... ] file.md ...
+
+Options:
+    -o, --output=PATH           Relative path of the output file
+    -i, --index                 Indicates that this is an index file
+    -h, --help                  Output this usage and do nothing else
+
+In usage 1, the template is read from standard input and the processing
+result is output to the path given by --output.  In this usage, the --output
+option is mandatory.
+
+In usage 2, the templates are read from the files given as argument, and
+the processing result for each of them is written to a corresponding file
+without the '.md' suffix.  All given file names must have the '.md' suffix.
```

Comment:
Perhaps add that -o is not allowed in this usage.

Reply:
Fixed